generate_markdown_from_doxygen_xml.py

```diff
@@ -116,7 +120,8 @@ def markdown_any_tag(aTag, html=False,para=True,consume=False):
             print('Unhanded tag in markdown_any_tag: %s' % child.tag)
             child_text+=str(ET.tostring(child),'utf-8')
 
-    # He we handle this tag.
+    # Here we handle this tag.
```

Comment:
I don't think it's clear what `this` means here.

Reply:
Fixed!